packages/list/ListItem.tsx

```diff
@@ -35,6 +35,7 @@ export interface ListItemProps<T> extends React.HTMLProps<T> {
   activated?: boolean;
   selected?: boolean;
   onDestroy?: () => void;
+  ref?: React.Ref<any>;
```

Comment:
Why did this change?

Reply:
ah good catch! This is actually a change from a future PR. This shouldn't be in here.